lib/logcat.js

```diff
@@ -106,6 +114,17 @@ class Logcat {
   getAllLogs () {
     return this.logs;
   }
+
+  async clear () {
+    log.debug('Clearing logcat logs from device');
+    try {
+      const args = this.adb.defaultArgs.concat(['logcat', '-c']);
```

Comment:
how about `args = [...this.adb.defaultArgs, 'logcat', '-c']` ?

Reply:
That's not generally how we've done it in this module. Even if it is preferable (not sure about that, though) it would still be better to keep consistent.

Reply:
I like spread operator, because it works for any iterable object, unlike concat, which is only the case for arrays.